pytest_services/xvfb.py

```diff
@@ -1,13 +1,17 @@
 """Fixtures for the GUI environment."""
 import os
-import fcntl
 import socket
 import re
 try:
     import subprocess32 as subprocess
 except ImportError:  # pragma: no cover
     import subprocess
 
+try:
+    import fcntl
```

Comment:
remove this import completely as it wasn't used

Reply:
umm. [line 74?](https://github.com/pytest-dev/pytest-services/blob/6d5551c09a11ea2bef36bc21924a9581d733aee4/pytest_services/xvfb.py#L74)